HHS/Head-Start-TTADP
# [TTAHUB-2811] Resolve XSS

## Description of change
This is a redo of @nvms's #2089. That had a messy history with most of it being reverted due to a bug introduced prior (the bug this very PR was intended to fix).

Use DOMPurify ([github page](https://github.com/cure53/DOMPurify)) ([npm page](https://www.npmjs.com/package/dompurify)) to sanitize rich fields.

## How to test
Put a bunch of formatted HTML into one of the rich text fields. Save to the database. Ensure that the stored data is completely sanitized. Reload the rich text field, and validate that what is returned to the client doesn't include &lt;, &gt;, etc.

## Issue(s)

* https://jira.acf.gov/browse/TTAHUB-2811


## Checklists

### Every PR



### Before merge to main


### Production Deploy


### After merge/deploy
